plasmapy/particles/ionization_state.py

```diff
@@ -123,7 +123,7 @@ def ionic_fraction(self) -> Real:
         An ionic fraction must be in the interval :math:`[0, 1]`.
 
         If no ionic fraction is specified, then this attribute will be
-        assigned the value of `~numpy.nan`.
+        assigned the value of |nan|.
```

Comment:
There's room for [a delicious glossary easter egg](https://en.wikipedia.org/wiki/Naan) here.